### PR TITLE
Ensure headings use consistent capitalisation

### LINKED ISSUE #612: Ensure headings use consistent capitalisation

### PR DESCRIPTION
**Pull request by courtneycb**
Resolves #612

### CODE REVIEW

**Comment by eAlasdair** [comment]
The branch in this PR has been brought up to date and appears correct for the changes made. However, any new content since September hasn't yet been considered. Whether or not we need to go back through it all I'm not sure

**Comment by eAlasdair** [comment]
This is not inclusive of all content but can still be merged when reviewed properly

File: csfieldguide/chapters/content/en/algorithms/sections/further-reading.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 - There is another searching algorithm which performs even better than Binary Search. It is called Hashing and can be investigated with the CS Unplugged [Battleships Game](http://csunplugged.org/searching-algorithms).
```
Comment (eAlasdair) [inline csfieldguide/chapters/content/en/algorithms/sections/further-reading.md]: binary search

File: csfieldguide/chapters/content/en/algorithms/sections/searching.md [hunk 2]
```diff
@@ -87,8 +88,10 @@ The binary search algorithm can be demonstrated with a phone book or dictionary:
...
 Throw away the half that can't contain the name, pointing out that hundreds of pages have been eliminated by one decision.
-Repeat this on the remaining half, ripping that in half, then half again, and so on. On the board you can work out the number of pages left; for example, if there were 512 pages in the phone book, after the first rip there are 256, then 128, then 64, 32, 16, 89, 4, 2 and finally just one page.
-That's  9 pages that were examined to get down to the desired page. (Note that it's easiest to pick numbers that are powers of 2 i.e. 512, 1024, 2048, otherwise you have to deal with halving odd numbers, which works fine, but is a bit distracting).
+Repeat this on the remaining half, ripping that in half, then half again, and so on.
+On the board you can work out the number of pages left; for example, if there were 512 pages in the phone book, after the first rip there are 256, then 128, then 64, 32, 16, 89, 4, 2 and finally just one page.
+That's  9 pages that were examined to get down to the desired page.
```
Comment (eAlasdair) [inline csfieldguide/chapters/content/en/algorithms/sections/searching.md]: unrelated but there's a double space here

File: csfieldguide/chapters/content/en/algorithms/sections/sorting.md [hunk 3]
```diff
@@ -133,7 +133,7 @@ For example if you owned a library and purchased a new book you wouldn't do a Se
 
 ## Quicksort
 
-Insertion and Selection Sort may seem like logical ways to sort things into order, but they both take far too many comparisons when they are used for large amounts of data.
+Insertion and selection sort may seem like logical ways to sort things into order, but they both take far too many comparisons when they are used for large amounts of data.
 Remember computers often have to search through HUGE amounts of data, so even if they use a good searching algorithm like Binary Search to look through their data, if they use a bad sorting algorithm to first sort that data into order then finding anything will take far too long!
```
Comment (eAlasdair) [inline csfieldguide/chapters/content/en/algorithms/sections/sorting.md]: binary search

File: csfieldguide/chapters/content/en/algorithms/sections/sorting.md [hunk 4]
```diff
@@ -133,7 +133,7 @@ For example if you owned a library and purchased a new book you wouldn't do a Se
...
 
 A much better sorting algorithm is {glossary-link term="quicksort"}Quicksort{glossary-link end}! (the name is a bit of a giveaway)
```
Comment (eAlasdair) [inline csfieldguide/chapters/content/en/algorithms/sections/sorting.md]: quicksort

File: csfieldguide/chapters/content/en/coding-compression/sections/run-length-encoding.md [hunk 5]
```diff
@@ -207,11 +207,11 @@ This image is from the [CS Unplugged image representation activity](http://csunp
 
 {panel end}
 
-The following interactive allows you to experiment further with Run Length Encoding.
+The following interactive allows you to experiment further with run length encoding.
 
 {interactive slug="run-length-encoding" type="whole-page" text="Run Length Encoding interactive"}
```
Comment (eAlasdair) [inline csfieldguide/chapters/content/en/coding-compression/sections/run-length-encoding.md]: 'text' doesn't work this way anyways, but: run length encoding interactive

**Comment by eAlasdair** [comment]
I've now changed enough that I probably can't review it properly, @courtneycb do you want to have another look?

**Comment by courtneycb** [comment]
> I've now changed enough that I probably can't review it properly, @courtneycb do you want to have another look?

sure

**Comment by courtneycb** [comment]
Happy with your changes @eAlasdair

**Pull request merged**